src/main/kotlin/org/rust/cargo/runconfig/target/RsLanguageRuntimeConfiguration.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig.target
+
+import com.intellij.execution.target.LanguageRuntimeConfiguration
+import com.intellij.openapi.components.BaseState
+import com.intellij.openapi.components.PersistentStateComponent
+
+class RsLanguageRuntimeConfiguration : LanguageRuntimeConfiguration(RsLanguageRuntimeType.TYPE_ID),
+                                       PersistentStateComponent<RsLanguageRuntimeConfiguration.MyState> {
```

Comment:
[minor] `MyState` can be imported explicitly

Reply:
We usually don't do it, because Kotlin suggest to add `RsLanguageRuntimeConfiguration` here.